.github/workflows/development_infrastructure.yml

```diff
@@ -1478,7 +1478,8 @@ jobs:
                 WEBSITES_PORT=5000 \
                 RATE_LIMIT_THRESHOLD=${{ vars.RATE_LIMIT_THRESHOLD }} \
                 PORTAL_REPORTING_PERIOD_START_EMAIL_SCHEDULE=${{ vars.PORTAL_REPORTING_PERIOD_START_EMAIL_SCHEDULE }} \
-                PORTAL_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH=${{ vars.PORTAL_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH }}
+                PORTAL_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH=${{ vars.PORTAL_REPORT_DUE_DATE_BUSINESS_DAYS_FROM_START_OF_MONTH }} \
```

Comment:
Can we call this UTILISATION_REPORT_DUE_DATE.... instead please? I can imagine we want to commonise this variable in TFM at some point when we are saying when stuff is due
Sorry this will affect the bicep PR too!

Reply:
Have updated all the new env vars to being `UTILISATION_REPORT...` 👍